src/iopae.cpp

```diff
@@ -233,7 +233,7 @@ void PaeInput::parsePlainAndEasy(std::istream &infile)
         }
 
         // rhythmic values
-        else if (isdigit(incipit[i]) != 0) {
+        else if ((isdigit(incipit[i]) != 0) && (incipit[i-1] != ';')) {
```

Comment:
Make sure i < 0

Reply:
C++ doesn't actually care.

Reply:
I do ;-) Sorry, I meant make sure `i > 0`

Reply:
Thanks. Just to make it clear: C++ does care.